data/human/navy missions intro.txt

```diff
@@ -0,0 +1,1477 @@
+				goto end
+			label next
+			branch help
+				has "rescue"
+			`	"No real reason. Just curious."`
+			`With some slight confusion, you just shrug and continue to wait while the rest of the mechanics are loaded aboard the transports.`
+				goto end
+			label help
+			apply
+				set "plan"
+				log "people" "Henry Radcliff" `An associate of Lucas Hadley, Radcliff and Hadley go way back. The two are practically brothers, and have vowed to keep an eye out for each other if anything went sideways. He also vows to be a private informant in case he sees anything sketchy.`
+			`	His expression suddenly softens. "The name's Henry Radcliff. I need to talk to you in private, if you don't mind." You nod, then turn to the other auxiliary captains. "I'll handle this one," you say to them.`
+			`Back aboard your ship, you lock the doors to your quarters before you and Radcliff sit down. "I'm glad I found you, Lieutenant," he begins. "Ever since we both started working for the Syndicate, Lucas and I made a promise to watch each others' backs. And it's my understanding that you made a promise to him, as well?"`
+			`	You nod. "When I learned that one of our officers had threatened Hadley, I wasn't about to let something like that slide."`
+			`	"And I thank you for that - there's no way I could bring that to my boss, and anyone else would just ignore me. But with you here, I think we have a real chance at getting Hadley and his family to safety."`
+			`	"So what's the plan?"`
+			`Henry procures a small holo-projector. "When the bombings happened, Lucas and I knew we had to come up with a contingency in case things got ugly. After considering our options, we decided to select this place." The projector highlights Hermes in the Capella System.`
+			`	"Hermes?" you say, with a sly grin. "That makes perfect sense. The planet's got orbital bombardment stations trained on the planet around the clock - if someone tried to tail you there, it's almost certain they'd be atomized by the relays."`
+			`	"That's exactly what we were thinking. And it's still close enough to Syndicate Space that it wouldn't be that far of a trip. Ideally, we wanted to settle in the Deep, but that's too far away."`
+			`The pair of you stand up. "I'll be sure to relay this to my superior officer, Henry," you reply, shaking his hand. "He's been trying to come up with a plan to get Hadley to safety, and this will be the break he needs."`
+			`	"I want to thank you once again, Lieutenant...?"`
+			`	"<last>. <first> <last>, if you want, Henry."`
+			`	"Thank you, <first>. Really, I can't thank you enough."`
+			`A thought enters your head. "Henry...I know this might be asking a lot, but if it's possible, I have a favor to ask. I'm sure you're well-aware of the bombings of Geminus and Martini at this point - if you get any leads, can I ask you to be my eyes and ears?"`
```

Comment:
Why is the player now investigating the bombings?

Did I miss something where it's called into doubt as to who did them in this storyline?  As far as I know, the Republic/Navy assumes it's the FW who did it, until evidence is presented.